ui/tests/__tests__/FacetCard.test.js

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import FacetCard from "../../src/components/facets/FacetCard";
+import { getFacet } from "../mockutils";
+import { Checkbox, MuiThemeProvider } from "material-ui";
+import { getMuiTheme } from "material-ui/styles";
+import PropTypes from "prop-types";
+
+test("Renders correctly with input", () => {
+  const tree = shallow(
+    <FacetCard facet={getFacet("Facet Name", 3)} updateFacet={() => {}} />
+  );
+
+  expect(tree).toMatchSnapshot();
+  expect(tree.find(".listItem").length).toBe(3);
```

Comment:
Isn't this line and next line covered by snapshot?

(As you know I prefer to only have these two lines and not snapshot, but if we're going to have snapshot, we don't need these.)